configs/arc64.config

```diff
@@ -0,0 +1,54 @@
+CT_CONFIG_VERSION="3"
+CT_EXPERIMENTAL=y
+CT_LOCAL_TARBALLS_DIR="${CT_PREFIX:-${HOME}/x-tools}/sources"
+# CT_PREFIX_DIR_RO is not set
+CT_PATCH_BUNDLED_LOCAL=y
+CT_LOCAL_PATCH_DIR="${CT_TOP_DIR}/../../patches"
+# CT_LOG_PROGRESS_BAR is not set
+CT_ARCH_ARC=y
+CT_ARCH_64=y
+CT_TARGET_VENDOR="zephyr"
+CT_BINUTILS_SRC_DEVEL=y
+CT_BINUTILS_DEVEL_URL="https://github.com/foss-for-synopsys-dwc-arc-processors/binutils-gdb.git"
+CT_BINUTILS_DEVEL_REVISION="9742c8bf3899b833122ce0bfbf3e112132c907cb"
```

Comment:
We need a set of patches added to the zephyr repo instead of git references like this.

Reply:
Are you sure? We use GCC 10.x releases in Zephyr's SDK, while ARC64 is based on GCC `master` branch (i.e. what's going to be GCC 11.x at some point). Imagine amount of changes involved (it's not just ARC code).

Moreover, from time to time we re-base our ARC64-related changes on top of upstream new `master` state and so those bumps will look like yet another huge patch for existing patch. So I'd prefer to use that much cleaner approach until our changes are merged upstream (say in a year or so).

Reply:
So for gcc we need to identify a baseline snapshot of the gcc11 work and than have any additional ARC changes on top of that.